### DISCUSSION TITLE
no veo el codigo en tiempo real en el navegador

### DISCUSSION THREAD

**Discussion by o327j** [community/community#142568] [Codespaces]
### Select Topic Area

General

### Body

si no puedo visualizar mi código en tiempo real con el navegador que debería realizar para que funcione la visualización del código en un navegador

**Comment by hamidconrade** [comment]
@o327j Para visualizar el código en tiempo real en un explorador, puede usar la extensión Live Server en Visual Studio Code. A continuación, te explicamos cómo hacerlo:

Instalar Live Server: Vaya a la pestaña Extensiones de Visual Studio Code y busque "Live Server". Instálalo.

Abra su archivo HTML: Abra el archivo HTML que desea previsualizar.

Iniciar servidor en vivo: Haga clic con el botón derecho en el archivo HTML y seleccione "Abrir con servidor en vivo". Esto iniciará un servidor local y abrirá su archivo HTML en su navegador predeterminado.

Ver cambios en tiempo real: Cualquier cambio que realice en el archivo HTML se actualizará automáticamente en el navegador.

¿Desea pasos más detallados o ayuda con algo más?

**Comment by Akash1134** [comment]
Hi @o327j , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
